src/features/category/categorySlice.ts

```diff
@@ -0,0 +1,58 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
```

Comment:
This file has lot of new learning points. Good work @sushil-mod . Do remove the debuggers mentioned in the thunks

Reply:
thanks, still a lot of things also I have to learn